appjs/views/application.js

```diff
@@ -135,9 +146,15 @@ var Application = BaseView.extend(require('./mixins/links.js'), {
     }
   },
 
+  toggleNav: function(event){
+    logger.debug('toggleNav', event);
+    if($('#navbar-collapse').hasClass('in')){
+      $('#navbar-collapse').collapse('toggle');
+    }
+  },
```

Comment:
Needed in order to get the mobile nav bar to retract correctly after a page has been selected.